package/boot/uboot-envtools/files/ipq806x

```diff
@@ -56,13 +56,17 @@ qcom,ipq8064-ap148|\
 qcom,ipq8064-db149)
 	ubootenv_add_uci_config $(ubootenv_mtdinfo)
 	;;
+xiaomi,mi-router-hd)
+	ubootenv_add_uci_config "/dev/mtd9" "0x0" "0x10000" "0x20000"
+	ubootenv_add_uci_sys_config "/dev/mtd12" "0x0" "0x10000" "0x20000"
+	;;
 ubnt,unifi-ac-hd|\
 zyxel,nbg6817)
 	ubootenv_add_uci_config "/dev/mtdblock9" "0x0" "0x10000" "0x10000"
 	;;
 esac
 
 config_load ubootenv
-config_foreach ubootenv_add_app_config ubootenv
```

Comment:
Unrelated change

Reply:
Let me remind you about this fix: https://github.com/openwrt/openwrt/commit/4cccea02a60aee0dd77c4db35672c92e2fe384a1

Without this patch, the `ubootenv_add_uci_sys_config` settings branch is simply ignored.